### PR TITLE
Box component + tweak ExpandBox

### PR DESCRIPTION
**Pull request by quietbits**
- Box component to use anywhere a container with a row gap is needed (`flex-direction: column`)
- Update `ExpandBox` to handle top offset (when collapsed, it was creating a double gap)

### CODE REVIEW

File: src/app/(sidebar)/account/create/page.tsx [hunk 1]
```diff
@@ -52,13 +52,14 @@ export default function CreateAccount() {
               Fund account with Friendbot
             </Button>
           </div>
+
+          <ExpandBox isExpanded={Boolean(account.publicKey)} offsetTop="xl">
+            <GenerateKeypair
+              publicKey={account.publicKey}
+              secretKey={secretKey}
+            />
+          </ExpandBox>
```
Comment (quietbits) [inline src/app/(sidebar)/account/create/page.tsx:61]: We can update the Fund Account page and other places if this looks good.

File: src/components/layout/Box/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export const Box = ({
+  gap,
+  children,
+  customValue,
+  addlClassName,
+}: (
+  | { gap: "xs" | "sm" | "md" | "lg" | "xl" | "xxl"; customValue?: undefined }
+  | { gap: "custom"; customValue: string }
+) & { children: React.ReactElement; addlClassName?: string }) => {
```
Comment (jeesunikim) [inline src/components/layout/Box/index.tsx:12]: I think `l` is a typo. It shold be `addClassName`?

**Review by jeesunikim — APPROVED** [review]
added a comment on typo but looks good. let's merge it!

File: src/components/layout/Box/index.tsx (hunk 2 above)
Comment (quietbits) [reply src/components/layout/Box/index.tsx:12]: It's actually "additional classes". We went with that convention in the SDS ([example](https://github.com/stellar/stellar-design-system/blob/sds-v3/%40stellar/design-system/src/components/Link/index.tsx#L27)). Thanks for checking!

**Pull request merged**